playground/env.prod-us.json

```diff
@@ -1,5 +1,6 @@
 {
   "frontendHost": "mc.commercetools.co",
+  "mcAuthUrl": "https://mc.commercetools.co",
```

Comment:
Thanks. I wanted to do this after I saw that we have an `ctpAuthUrl` as an env variable.